Rakefile

```diff
@@ -25,8 +25,12 @@ if defined?(RSpec) && defined?(RSpec::Core::RakeTask)
   end
 end
 
-require 'jasmine'
-load 'jasmine/tasks/jasmine.rake'
+# Only load the jasmine tasks if we are within this repo, otherwise, the bundle
+# won't contain the jasmine gem (i.e., from manageiq)
+if ENV["BUNDLE_GEMFILE"].nil? || ENV["BUNDLE_GEMFILE"] == File.expand_path("../Gemfile", __FILE__)
```

Comment:
I've never been a fan of using `__FILE__` or any other "files" in the "relative from" parameter to expand_path.  It's so much clearer with directories.

```ruby
File.expand_path("./Gemfile", __dir___)
```

...is so much clearer IMO.